SpatialGDK/Source/SpatialGDK/Private/Utils/EntityFactory.cpp

```diff
@@ -61,33 +59,23 @@ TArray<FWorkerComponentData> EntityFactory::CreateEntityComponents(USpatialActor
 	WorkerRequirementSet AnyServerOrOwningClientRequirementSet = { OwningClientAttributeSet };
 	WorkerRequirementSet OwningClientOnlyRequirementSet = { OwningClientAttributeSet };
 
-	for (const FName& WorkerType : GetDefault<USpatialGDKSettings>()->ServerWorkerTypes)
-	{
-		WorkerAttributeSet ServerWorkerAttributeSet = { WorkerType.ToString() };
+	WorkerAttributeSet ServerWorkerAttributeSet = { SpatialConstants::DefaultServerWorkerType.ToString() };
```

Comment:
Can we move this up to where the `ClientWorkerAttribute` is defined? Then we could define a lot of the requirement sets inline instead of having the `.Add(` calls

Reply:
Good idea. Done.